src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGenerator.java

```diff
@@ -70,8 +69,14 @@ public String prepareMailContent(int numberOfDaysBackInThePast) {
         String templateContent =  mailingTemplate.getValue();
         StringTemplate template = new StringTemplate(templateContent);
         template.setAttribute("days", numberOfDaysBackInThePast);
-        template.setAttribute("newPosts", newPostsFromPersonalBlogs.stream().map(BlogPostForMailItem::new).collect(Collectors.toList()));
-        template.setAttribute("newPostsFromCompanies", newPostsfromCompanies.stream().map(BlogPostForMailItem::new).collect(Collectors.toList()));
+        template.setAttribute("newPosts", newPostsFromPersonalBlogs.stream().map(
```

Comment:
Duplicated code for "newPosts" and "newPostsFromCompanies". You could extract method like
```java
private List<BlogPostForMailItem> toMailItems(List<BlogPost> newPosts, long issueNumber) {
        return newPosts.stream().map(blogPost ->
                BlogPostForMailItem.builder().from(blogPost)
                        .withIssueNumber(issueNumber)
                        .withDefaultUTMParameters()
                        .build())
                .collect(Collectors.toList());
    }
```
and then call
```java
template.setAttribute("newPosts", toMailItems(newPostsFromPersonalBlogs, issueNumber))
```